spec/spec_helper.rb

```diff
@@ -4,7 +4,10 @@
 require "scorm_engine"
 
 # Ensure we're picking up only the test scorm settings
-Dotenv.load(".env.test")
+Dotenv.load(".env.test.local",
+            ".env.test",
+            ".env.local",
```

Comment:
Curious why you added the non-test env files here? Couldn't that cause settings I have in my env.local to override what's used when running specs locally?

Reply:
That is the same thing that is done in Learn.

Dotenv.load(".env.test.local",
            ".env.test",
            ".env.local",
            ".env")

Reply:
Yeah, I purposely mimicked what learn is doing, since I expect that behavior.

Reply:
I'm alright trying it this way. I'll just be careful when running the specs locally that I'm not bleeding in any other vars, which I can't imagine doing after the Engine migration 🤷

Reply:
The documentation says `By default, load will look for a file called .env in the current working directory. Pass in multiple files and they will be loaded in order. The first value set for a variable will win.`

1st priority == .env.test.local
2nd == .env.test
3rd == .env.local

Another way of looking at is like.. `.env.local.merge(.env.test).merge(.env.test.local)`

Reply:
Oh cool! That totally alleviates my concern then 👍